integr8ly/installation
# feat: Add RHSSO alerts

## Additional Information
Adding additional RHSSO alerts from the new version of the operator. These can be viewed here - https://github.com/integr8ly/keycloak-operator/pull/95/files

## Verification Steps
1. Install Integreatly from this PR branch
2. Login to Prometheus in the middleware-monitoring namespace.
3. Verify the alerts KeycloakInstanceNotAvailable, KeycloakAPIRequestDuration90PercThresholdExceeded, KeycloakAPIRequestDuration99PercThresholdExceeded, KeycloakDatabaseNotAvailable are present
4. Scale down the sso pod in the sso namespace
5. Wait 5 minutes and verify the alert triggers.
6. Scale back up the sso pod
7. Repeat steps 4-6 for the postgres pod
8. Modify the PrometheusRule for the 90% alerts `sso > resources > other-resources > PrometheusRule > application-monitoring`  and edit yaml. Change the `10000.0` value to `50.0` in the KeycloakAPIRequestDuration99PercThresholdExceeded alert.
9. Wait 5 minutes and verify the alert triggers.
10. The 90% alert does not need to be verified as it is a carbon copy of the other except for the bucket and percentage checked


## Is an upgrade task required and are there additional steps needed to test this?
1. Install release-1.5.0
3. Upgrade integreatly from this branch
3. Verify the alerts KeycloakInstanceNotAvailable, KeycloakAPIRequestDuration90PercThresholdExceeded, KeycloakAPIRequestDuration99PercThresholdExceeded, KeycloakDatabaseNotAvailable are present
